styles.css

```diff
@@ -193,6 +339,108 @@ ul {
   margin: 0 5px 0 5px;
 }
 
+.featured-stories-title {
+  color: #000;
+  font-size: 20px;
+}
+
+.featured-stories-text {
+  color: #333;
+  font-size: 12px;
+}
+
+/* Mobile */
+
+@media screen and (max-width: 420px) {
+  .ups {
+    left: 126px;
+  }
+}
```

Comment:
- [ ] Is a mobile-first approach, remove the media query and make it work with the correct aproach